### PR TITLE
Fleet qa/update cypress

### PR DESCRIPTION
**Pull request by mmartin24**
# Done
Implements: https://github.com/rancher/fleet-e2e/issues/44

- Updated Cypress to the latest version as of today: `13.6.4`
- Updated Cypress Docker images to: `cypress/included:cypress-13.6.4-node-20.11.0-chrome-121.0.6167.85-1-ff-120.0-edge-121.0.2277.83-1`
- Added uncaught exceptions to avoid crashes
- Added video flags as by default videos are not passed in this version.
- Removed command `overwrite` which is incompatible with cy 13. This has turned into a significant speed increase (see screenshot)
image (2)

# Tests passing:

- [UI-RM_head_2.7 #37](https://github.com/rancher/fleet-e2e/actions/runs/7927306819)
- [UI-RM_head_2.8 #73](https://github.com/rancher/fleet-e2e/actions/runs/7927130411)
- [UI-RM_head_2.9 #66](https://github.com/rancher/fleet-e2e/actions/runs/7927309676)
- 
image (3)

### CODE REVIEW

File: .github/workflows/master-e2e.yaml [hunk 1]
```diff
@@ -156,7 +156,8 @@ jobs:
       - name: Cypress tests - Basics
         env:
           BROWSER: chrome
-          CYPRESS_DOCKER: 'cypress/included:10.9.0'
+          # CYPRESS_DOCKER: 'cypress/included:10.9.0'
+          CYPRESS_DOCKER: 'cypress/included:cypress-13.6.4-node-20.11.0-chrome-121.0.6167.85-1-ff-120.0-edge-121.0.2277.83-1'
```
Comment (thehejik) [inline .github/workflows/master-e2e.yaml:160]: Please use short tag `:13.6.4` instead:
image

File: tests/cypress/package.json [hunk 2]
```diff
@@ -34,6 +33,7 @@
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "^6.2.0",
     "@typescript-eslint/parser": "^6.2.0",
+    "cypress": "^13.6.4",
```
Comment (thehejik) [inline tests/cypress/package.json:36]: Please move back to `dependencies:` - otherwise `--dev` might be needed for `npm install`

Comment (mmartin24) [reply tests/cypress/package.json]: thanks .done

File: .github/workflows/master-e2e.yaml (hunk 1 above)
Comment (mmartin24) [reply .github/workflows/master-e2e.yaml]: thanks. done.

**Comment by mmartin24** [comment]
tested again and working: https://github.com/rancher/fleet-e2e/actions/runs/7956760263/job/21718123606

**Pull request merged**